templates/case/_case_show/_resident_and_rent_board_shareds.html.twig

```diff
@@ -74,13 +74,13 @@
                             <input type="text" class="form-control" value="{{ case.complainantAddress.city }}" readonly>
                         </div>
                         <div class="form-group col-md-6">
-                            <label>{% trans %}Validate{% endtrans %}</label>
-                            {# TODO: Show status depending on validation result #}
-                            {# Not yet validated #} <button class="btn btn-block btn-secondary" type="button" data-toggle="tooltip" data-placement="auto" title="{{ 'Validate address'|trans([], 'case', 'da') }}"><i class="fas fa-check-circle"></i></button>
-                            {# Success <button class="btn btn-block btn-success" type="button" data-toggle="tooltip" data-placement="auto" title="Address is validated"><i class="fas fa-check-circle"></i></button>#}
-                            {# Warning <button class="btn btn-block btn-warning" type="button" data-toggle="tooltip" data-placement="auto" title="Warning could not validate"><i class="fas fa-exclamation-circle"></i></button>#}
-                            {# Failed <button class="btn btn-block btn-danger" type="button" data-toggle="tooltip" data-placement="auto" title="Error validating"><i class="fas fa-times-circle"></i></button>#}
-                            <p class="text-muted">{% trans %}Validate the legitimacy of the address by pressing the validate button{% endtrans %}</p>
+                            <label class="invisible">{% trans %}Validate{% endtrans %}</label>
+                            {% if case.complainantAddress.validatedAt %}
+                                <span class="btn btn-block btn-success" type="button" data-toggle="tooltip" data-placement="auto" title="{{ 'Address validated at %validation_time%'|trans({'%validation_time%': case.complainantAddress.validatedAt|date}) }}"><i class="fas fa-check-circle"></i> {% trans %}Address validated{% endtrans %}</span>
+                            {% else %}
+                                {# Render ajax form #}
+                                {{ render(controller('App\\Controller\\CaseController:validateAddress', {id: case.id, addressProperty: 'complainantAddress'})) }}
+                            {% endif %}
```

Comment:
We probably want to do the same in `fence_review_case.html.twig` with both complainant and counterpart.

Reply:
Fixed in https://github.com/itk-dev/naevnssekretariatet/pull/78/commits/8d3245959e9c1050b3e392aaadf95e10a51af1ce